src/app/profile/user/details/details.component.html

```diff
@@ -7,13 +7,24 @@
         <div class="container">
           <div class="profile-header-info__user-card">
             <div class="profile-header-info__photo">
-              <cl-image height="150" crop="scale" public-id="{{ user.profile_pic }}" secure="true"></cl-image>
+              <cl-image
+                height="150"
```

Comment:
Please remove this auto-linting, else it will change multiple time to and fro.

Reply:
Can you please help me understand auto-linting? I use visual studio code, so should I disable something on it or do I need to change the code in details.component.html?